src/main/java/uk/gov/hmcts/cmc/claimstore/documents/content/FullDefenceResponseContentProvider.java

```diff
@@ -46,6 +47,10 @@
             content.put("paymentDeclaration", createContentFor(paymentDeclaration))
         );
 
+        fullDefenceResponse.getFreeMediation().ifPresent(mediation ->
+            content.put("mediation", mediation.equals(YesNoOption.YES))
```

Comment:
This puts `true` as the value, but from the other code in this PR it looks like you're expecting `yes` and `no`.
Try `content.put("mediation", mediation.name().toLowerCase())`

Reply:
Just discussed with @KiranVarma02 . Template expects true or false. The yes or no is determined in template.

Reply:
`mediation == YesNoOption.YES` would be preferred instead of equals here